src/explore-education-statistics-common/src/modules/charts/components/MapBlockInternal.tsx

```diff
@@ -59,6 +59,7 @@ export interface MapBlockProps extends ChartProps {
   axes: {
     major: AxisConfiguration;
   };
+  // eslint-disable-next-line react/no-unused-prop-types
```

Comment:
Don't think we actually need this prop, however, we'll need to adjust `ChartRenderer` props i.e.

```ts
export type ChartRendererProps = {
  source?: string;
} & (
  | ({
      type: 'line';
    } & LineChartProps)
  | ({
      type: 'verticalbar';
    } & VerticalBarProps)
  | ({
      type: 'horizontalbar';
    } & HorizontalBarProps)
  | ({
      type: 'map';
    } & Omit<MapBlockProps, 'id'> & {
        boundaryLevel?: number;
      })
  | ({
      type: 'infographic';
    } & InfographicChartProps)
);
```